source/release-notes/release_3_13_0.rst

```diff
@@ -23,34 +24,34 @@ Wazuh core
 - Added a timeout to the updates of the vulnerability detector's feeds to prevent hangings.
 - Added option for the JSON decoder to choose the treatment of array structures.
 - Added ``mode`` value (real-time, Who-data, or scheduled) as a dynamic field in FIM alerts. 
-- Added a configurable maximum limit of files to be monitored by FIM.
-- New integration for pull logs from Google Cloud Pub/Sub.
-- Added support for MITRE ATT&CK knowledge base.
+- Added a field to configure the maximum files to be monitored by the FIM.
+- Added a module to integrate Google Cloud Pub/Sub.
+- Added support for mapping Wazuh rules with MITRE ATT&CK knowledge base.
 - Added new configuration block for Windows Defender EventChannel.
```

Comment:
Please remove this, it was finally removed (Windows Defender configuration block).

Reply:
Done